src/components/Pages/Guide/_Guide.scss

```diff
@@ -259,6 +259,10 @@ $max-guide-tablet-width: $coa-medium-screen;
   width: 100%;
 }
 
+.coa-GuidePage__content-container > .wrapper > .row {
```

Comment:
it is strange that this was getting set to -1 before, probably would be nice to figure out where our bootstrappy things are causing wonkyness/one-offy css.

Reply:
I found where someone, at sometime, added this note. I've added a reference to that `TODO` within the css as well. 
https://github.com/cityofaustin/janis/blob/0cba7da41b4d6b637becb7b05f73255104a37528/src/components/Pages/Guide/_Guide.scss#L72